gel/_internal/_qbmodel/_pydantic/_models.py

```diff
@@ -383,6 +389,24 @@ def __delattr__(self, name: str) -> None:
         )
 
 
+def _kwargs_exclude_id(
+    id_: uuid.UUID, kwargs: dict[str, Any], /
+) -> dict[str, Any]:
+    if id_ is UNSET_UUID:
+        try:
+            exclude = kwargs["exclude"]
+        except KeyError:
+            exclude = kwargs["exclude"] = set()
+
+        if isinstance(exclude, set):
+            exclude.add("id")
+        else:
+            assert isinstance(exclude, dict)
+            exclude["id"] = True
```

Comment:
Slightly more optimal, since "exclude" will not be there most of the time:

```suggestion
        try:
            exclude = kwargs["exclude"]
        except KeyError:
            kwargs["exclude"] = {"id"}
        else:
            if isinstance(exclude, set):
                exclude.add("id")
            else:
                assert isinstance(exclude, dict)
                exclude["id"] = True
```

Reply:
++